packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -430,6 +430,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
           highlightedIndex={highlightedIndex}
           onStateChange={this.handleStateChange}
           labelId={this.props['aria-labelledby']}
+          environment={this.context.target.defaultView}
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Document/defaultView
Works in IE11, I am happy 👍

Reply:
the exact place I got it from!